bin/lib.sh

```diff
@@ -436,7 +437,7 @@ show_spinner_pid() {
     fi
     local temp
     tput civis
-    while [[ -e /proc/$pid ]]; do
+    while ps a | awk '{print $1}' | grep -q "${pid}"; do
```

Comment:
this work right on osx?

Reply:
Yeah this is actually reverting back to an earlier commit. I tried to simplify it to `/proc/$pid`, but OS X does not have procfs, so you have to do this hack-yness instead.